README.md

```diff
@@ -299,6 +300,27 @@ To deep down the `buildLocalReference` arguments, you may read the [documentatio
 <a name="route.options"></a>
 ### Route options
 
+It is possible to instruct `@fastify/swagger` to include specific `HEAD` routes in the definitions
+by adding `exposeHeadRoute` to the route schema definition, like so:
+
+```js
+  fastify.get('/with-head', {
+    schema: {
+      operationId: 'with-head',
+      exposeHeadRoute: true,
```

Comment:
The name is same as route option, would it be too difficult for the user to distinguish two of them?

Reply:
what would you recommend instead?

Reply:
I am thinking should we utilize `routeOption.config`? or simple `exposeHeadToSwagger`?

```js
fastify.get('with-head', {
  schema: {
    // ...
  },
  config: {
    swagger: {
      exposeHeadRoute: true
    }
  }
})
```